pom.xml

```diff
@@ -49,10 +49,12 @@
         <commons-httpclient.version>3.0.1</commons-httpclient.version>
         <dep.hadoop1.version>1.2.0</dep.hadoop1.version>
         <dep.hadoop2.version>2.6.0</dep.hadoop2.version>
+        <dep.dropwizard.version>3.1.0</dep.dropwizard.version>
     </properties>
 
     <modules>
         <module>rubix-spi</module>
+        <module>rubix-common</module>
```

Comment:
Another generic comment. Can you add a new page in docs to document these metrics ? Similar to http://docs.qubole.com/en/latest/user-guide/presto/presto-system-metrics.html

This can be a basic doc page right now that just list the metrics. We can build it out incrementally. I suggest you start a new section - `Admin Guide`(Right now there are `Rubix` and `Installation Guide` sections). Under that start a sub-section `Metrics`. With this PR, it should contain a list of metrics you've added. No need of descriptions etc right now. Also put a warning that the page is under construction in parentheses.

Reply:
Should we document these on the repository's wiki as well?

Reply:
We are going to update that as well